test-integration/helpers/ui/reference-book.coffee

```diff
@@ -35,6 +35,16 @@ class ReferenceBook extends TestHelper
     @test.utils.windowPosition.setLarge()
     @el.tocToggle().click()
 
+  focus: =>
+    @test.driver.getAllWindowHandles().then (handles) =>
+      @test.driver.switchTo().window(handles[1])
```

Comment:
Nice! much better than opening the URL directly (plus it verifies the click worked)